src/Dev/Install/Installer.php

```diff
@@ -55,8 +55,8 @@ protected function error($message = null)
     protected function installHeader()
     {
         $clientPath = PUBLIC_DIR
-            ? 'resources/vendor/silverstripe/framework/src/Dev/Install/client'
-            : 'resources/silverstripe/framework/src/Dev/Install/client';
+            ? '_resources/vendor/silverstripe/framework/src/Dev/Install/client'
```

Comment:
Why not use the constant for these bits?

Reply:
Laziness.